platform/features/listview/src/controllers/ListViewController.js

```diff
@@ -0,0 +1,40 @@
+define(function () {
+  function ListViewController($scope){
+      this.$scope = $scope;
+    $scope.orderByField = 'title';
+    $scope.reverseSort = false;
+
+    this.updateView()
+
+    this.unlisten = $scope.domainObject.getCapability('mutation')
```

Comment:
That's the correct way to remove the mutation listener, but where is this `model` function called from?